process-engine/bpmn-studio
# 🐛 Fix handling of TimerStartEvents

## Changes

Fix an issue with the ProcessEngineRuntime, which prevented a user from stopping ProcessInstances that start with a `TimerStartEvent`.

## Issues

Closes #1461

PR: #1465

## How to test the changes

1. Try starting ProcessModels that contain the various types of TimerStartEvents (Duration, Date, Cyclic)
2. The following UseCases should produce an error right at the beginning:
    - A cyclic timer is used (these are currently not supported)
    - A past date is used with a Date-timer
    - An invalid format is used for a Duration-timer
3. Try stopping a ProcessInstance that started successfully
4. You'll see that the ProcessInstance gets finished properly now
